sources/packages/backend/apps/workers/src/controllers/supporting-user/supporting-user.controller.ts

```diff
@@ -172,20 +172,53 @@ export class SupportingUserController {
           return job.error(SUPPORTING_USER_FULL_NAME_NOT_RESOLVED, message);
         }
       }
+      const isAbleToReport = job.variables.isAbleToReport;
+      // If the full name is not provided, use the student
       return this.dataSource.transaction(
-        // Transaction created with the expectation that a notification will also be created
-        // for the supporting user and it should be saved in the same transaction.
         async (entityManager: EntityManager) => {
           const createdSupportingUserId =
             await this.supportingUserService.createIdentifiableSupportingUser(
               {
                 applicationId: job.variables.applicationId,
                 supportingUserType: job.variables.supportingUserType,
                 fullName,
-                isAbleToReport: job.variables.isAbleToReport,
+                isAbleToReport,
               },
               entityManager,
             );
+          if (isAbleToReport) {
+            await this.notificationActionsService.saveParentDeclarationRequiredParentCanReportNotification(
+              {
+                givenNames: application.student.user.firstName,
+                lastName: application.student.user.lastName,
+                toAddress: application.student.user.email,
+                userId: application.student.user.id,
+                supportingUserType:
+                  job.variables.supportingUserType === "Parent"
```

Comment:
As we have been following everywhere, please use the enum instead of string. `=== SupportingUserType.Parent`